src/flair/flair_transcriptome.py

```diff
@@ -998,6 +1176,79 @@ def combineannotnovelwriteout(args, genetojuncstoends, genome):
                     else:
                         endslist.sort(key=lambda x: [len(x[-1]), x[1] - x[0]], reverse=True)
                         endslist = endslist[:args.max_ends]
+                genetojuncstoends[gene][(chrom, strand, juncs)] = endslist
+
+
+        ###TESTED THIS - doesn't seem to actually have an impact, subsets are handled appropriately earlier
```

Comment:
please don't include commented-out code. If this is something you want to leave in, make it a function and then not call it.

It this isn't done, we shouldn't merge it yet.